### PR TITLE
Read config values from the environment

### PR DESCRIPTION
**Pull request by thatbudakguy**
This ensures the SETTINGS__THROTTLE_TRAFFIC envvar will take effect

### CODE REVIEW

File: config/initializers/rails_config.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
 Config.setup do |config|
   config.const_name = "Settings"
+  config.use_env = true
```
Comment (jcoyne) [inline config/initializers/rails_config.rb:3]: Do you also need to set:
```
config.env_separator = '__'
config.env_converter = :downcase
```

**Review by jcoyne — APPROVED** [review]
Do you a

File: config/initializers/rails_config.rb (hunk 1 above)
Comment (thatbudakguy) [reply config/initializers/rails_config.rb:3]: looks like `:downcase` is the default, but the default separator is `.`, so I do need to set that. good catch

Comment (thatbudakguy) [reply config/initializers/rails_config.rb:3]: updated

**Review by jcoyne — APPROVED** [review]

**Comment by thatbudakguy** [comment]
Using `shared_configs` instead per @cbeer: https://github.com/sul-dlss/shared_configs/pull/1951/files

**Pull request closed without merging**